### PR TITLE
[ja] Update concepts/cluster-administration/addons.md

### LINKED ISSUE #32999: [ja] `concepts/cluster-administration/addons` might be outdated
**This is a Bug Report**

**Problem:**
The file `content/ja/docs/concepts/cluster-administration/addons.md` might be outdated.
- en: https://kubernetes.io/docs/concepts/cluster-administration/addons/
  Under "Networking and Network Policy" section,
  the entry "Contiv" exists.
- ja: https://kubernetes.io/ja/docs/concepts/cluster-administration/addons/
  The entry "Contiv" does not exist.

**Proposed Solution:**
Update `content/ja/docs/concepts/cluster-administration/addons.md`.

**Page to Update:**
https://kubernetes.io/ja/docs/concepts/cluster-administration/addons/

/language ja

### PR DESCRIPTION
**Pull request by t-inu**
fixes #32999
Restore "Contiv" entry, and apply other en updates.

List of updates ([History](https://github.com/kubernetes/website/commits/main/content/en/docs/concepts/cluster-administration/addons.md))
- https://github.com/kubernetes/website/pull/29719
- https://github.com/kubernetes/website/pull/31760
  - https://github.com/kubernetes/website/pull/32938
- https://github.com/kubernetes/website/pull/32834
  - https://github.com/kubernetes/website/pull/32937
    - https://github.com/kubernetes/website/pull/34171
- https://github.com/kubernetes/website/pull/34126

en:
https://kubernetes.io/docs/concepts/cluster-administration/addons/
https://github.com/kubernetes/website/blob/main/content/en/docs/concepts/cluster-administration/addons.md

ja:
https://kubernetes.io/ja/docs/concepts/cluster-administration/addons/
https://github.com/kubernetes/website/blob/main/content/ja/docs/concepts/cluster-administration/addons.md

/assign @nasa9084

### CODE REVIEW

**Review by nasa9084 — COMMENTED** [review]
/approve

File: content/ja/docs/concepts/cluster-administration/addons.md [hunk 1]
```diff
@@ -44,6 +45,7 @@ content_type: concept
 ## インフラストラクチャ
 
 * [KubeVirt](https://kubevirt.io/user-guide/#/installation/installation)は仮想マシンをKubernetes上で実行するためのアドオンです。通常、ベアメタルのクラスタで実行します。
+* [node problem detector](https://github.com/kubernetes/node-problem-detector)はLinuxノード上で動作し、システムの問題を[Events](/docs/reference/kubernetes-api/cluster-resources/event-v1/)または[ノードのCondition](/ja/docs/concepts/architecture/nodes/#condition)として報告します。
```
Comment (s-kawamura-w664) [inline content/ja/docs/concepts/cluster-administration/addons.md:48]:
```suggestion
* [node problem detector](https://github.com/kubernetes/node-problem-detector)はLinuxノード上で動作し、システムの問題を[Event](/docs/reference/kubernetes-api/cluster-resources/event-v1/)または[ノードのCondition](/ja/docs/concepts/architecture/nodes/#condition)として報告します。
```

I think "Event" would be better as above.

1. The following is the localization rule. (https://kubernetes.io/ja/docs/contribute/localization/)
    ```
    また、日本語では名詞を複数形にする意味はあまりないので、
    英語の名詞を利用する場合は原則として単数形で表現してください。
    ```
3. Linked site's title is "Event".
4. In English's following site, "Event" is used.
    * https://github.com/kubernetes/website/blob/main/content/en/docs/concepts/configuration/manage-resources-containers.md#L200:~:text=be%20found.%20An-,Event%20is,-produced%20each%20time
    * https://github.com/kubernetes/website/blob/main/content/en/docs/reference/labels-annotations-taints/audit-annotations.md#:~:text=objects%20in%20the-,Event%20API,-(API%20group%20events

**Review by s-kawamura-w664 — COMMENTED** [review]
Almost no problem.
I commented on one point.

File: content/ja/docs/concepts/cluster-administration/addons.md (hunk 1 above)
Comment (t-inu) [reply content/ja/docs/concepts/cluster-administration/addons.md]: Thanks, fixed.

**Comment by s-kawamura-w664** [comment]
Thanks for the update. It is so good.
/lgtm

**Pull request merged**